spec/page/posts_spec.rb

```diff
@@ -1,24 +1,44 @@
 require 'spec_helper'
 
 RSpec.describe 'Fb::Page#posts' do
-  context 'given an invalid access token' do
-    let(:page) { Fb::Page.new access_token: 'invalid_token' }
-
-    it 'raises Fb::HTTPError' do
-      expect{page.posts}.to raise_error Fb::HTTPError
-    end
-  end
```

Comment:
@S1v4 the code above should not be affected by your current commit.
Can you leave it as it was?